okapi-core/src/main/raml/okapi.raml

```diff
@@ -666,6 +666,15 @@ types:
               text/plain:
       delete:
         description: Disable modules for tenant
+        queryParameters:
+          purge:
+            description: Disabled modules will also be purged.
+            type: boolean
+            required: false
+          tenantParameters:
+            description: Parameters for Tenant init
+            type: string
+            required: false
```

Comment:
Maybe add a purge test for this one too?

Reply:
Could do.. but that would mostly be to check that RAML is ok.. There's 100% coverage.